tox.ini

```diff
@@ -10,10 +10,4 @@ addopts = -v --cov rsa --cov-report term-missing
 deps = poetry
 commands =
     poetry install
-    poetry run py.test tests/
-
-[testenv:py37]
-whitelist_externals = poetry
-commands=
-    poetry install
-    poetry run py.test --doctest-modules rsa tests/
+    poetry run pytest tests/
```

Comment:
The doctest modules were run only in one version of Python, and not all of them, because the Python code produced subtly different output on different versions, and that made it impossible for the tests to be compatible with all Python versions.

The doctests still need to be run, so either add `--doctest-modules rsa` to this `poetry run pytest tests/` (if it works for all of 3.8-3.12) or just make sure the tests work for one version again (I'd recommend checking with 3.11 then, as that's the latest release version).